src/config/Config.cpp

```diff
@@ -226,6 +226,16 @@ AudioBackend Config::GetCurrentAudioBackend() {
     return AudioBackend::SDL;
 }
 
+AudioChannelsSetting Config::GetCurrentAudioSurround() {
+    int surround = GetInt("CVars.gAudioChannelsSetting");
```

Comment:
sorry i missed this one before, having this get use a hardcoded string will make it so ports changing the string for `CVAR_AUDIO_CHANNELS_SETTING` won't work.

```suggestion
    int surround = GetInt(CVAR_AUDIO_CHANNELS_SETTING);
```

also, we should probably have a default in here

```suggestion
    int surround = GetInt(CVAR_AUDIO_CHANNELS_SETTING, 0);
```

and we should probably avoid using magic numbers

```suggestion
    int32_t surround = GetInt(CVAR_AUDIO_CHANNELS_SETTING, static_cast<int32_t>(AudioChannelsSetting::stereo));
```

and we should probably not just rely on `stereo` being `0`

```suggestion
    int32_t surround = GetInt(CVAR_AUDIO_CHANNELS_SETTING, static_cast<int32_t>(AudioChannelsSetting::max));
```